Sources/Sentry/SentryFileManager.m

```diff
@@ -634,6 +634,11 @@ - (BOOL)isDirectory:(NSString *)path
     return [NSFileManager.defaultManager fileExistsAtPath:path isDirectory:&isDir] && isDir;
 }
 
+/**
+ * @note This method must be statically accessible because it will be called during app launch,
+ * before any instance of  ``SentryFileManager`` exists, and so wouldn't be able to access this path
+ * from an objc property on it like the other paths.
+ */
```

Comment:
Thanks for the comment.

Reply:
You are welcome. I actually got this from [this PR](https://github.com/getsentry/sentry-cocoa/commit/f8833c443d047f0c8dac5710e52b4c0624fc52ce#diff-98c5d4f2fdf1f254efbb66e82cbc8109f224038b61270c4886cce083c12664e3R732-R753), but not sure when it got lost